src/ClickRenderer.js

```diff
@@ -58,7 +60,12 @@ export default class ClickRenderer {
   onWindowSize(event) {
     clearTimeout(this.resizeTimeout)
     this.resizeTimeout = setTimeout(() => {
-      this.redrawClicks();
+      const docSize = document.querySelector('html').getBoundingClientRect()
+      this.canvas.width = Math.round(docSize.width)
+      this.canvas.height = Math.round(docSize.height)
+      requestAnimationFrame(() => {
```

Comment:
I would need to add a bind(this) to make this work, and the fat arrow function already accomplishes this. For personal style reasons I'm going to leave it how it is.